README.md

```diff
@@ -75,6 +75,15 @@ Example modules are available in [examples](examples) folder. You can use `cargo
 For example (all examples plus linux specific):
 `cargo build --package=examples --examples --features=linux`
 
+### Build with external NGINX source tree
+
+If you require a custom NGINX configuration, you can build a module against an existing pre-configured source tree.
+To do that, you need to set the `NGX_OBJS` variable to an _absolute_ path to the NGINX build directory (`--builddir`, defaults to the `objs` in the source directory).
+Only `./configure` step of the NGINX build is mandatory, as bindings don't depend on any of the artifacts generated by `make`.
+
+Furthermore, this could be leveraged to build a module as a part of the NGINX build process with `--add-module`/`--add-dynamic-module` options of the configure script.
+See example integration scripts at [`examples/config`](examples/config) and [`examples/config.make`](examples/config.make).
+
```

Comment:
```suggestion
### Build with external NGINX source tree

If you require a customized NGINX configuration, you can build a module against an existing pre-configured source tree.
To do that, you need to set the `NGX_OBJS` variable to an _absolute_ path of the NGINX build directory (`--builddir`, defaults to the `objs` in the source directory).
Only the `./configure` step of the NGINX build is mandatory because bindings don't depend on any of the artifacts generated by `make`.

Furthermore, this approach can be leveraged to build a module as a part of the NGINX build process by adding the `--add-module`/`--add-dynamic-module` options to the configure script.
See the following example integration scripts: [`examples/config`](examples/config) and [`examples/config.make`](examples/config.make).

```